controllers/nf/smf/resources_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
-func TestCreateDeployment(t *testing.T) {
+func CreateDeployment(t *testing.T) {
```

Comment:
I think that makes more sense to keep the `Test` prefix here

Reply:
Yes. Agreed. We can keep Test there.